deploy/role_binding_ui.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
I don't think we need a role binding. we just want to create a role that can be bound to other service-accounts by the user